index.html

```diff
@@ -18,6 +18,10 @@
     
 </head>
 <body>
+  
```

Comment:
this is not necessary. you should be able to draw a rectangle object within Phaser and assign a width and height equal to the browser (e.g. `this.game.width`).

check out the following for help:
```
var PausePanel = function(game, parent){
...
    var backgroundColor = "0xffffff";
    var backgroundOpacity = 0.7;

    var pause_bg = game.add.graphics(game.width, game.height);
    pause_bg.beginFill(backgroundColor, backgroundOpacity);
    pause_bg.x = 0;
    pause_bg.y = 0;

    pause_bg.drawRect(0, 0, game.width, game.height);

    this.add(pause_bg);
```